### ISSUE TITLE
Requested URL Not Found During Initial Setup

### ISSUE THREAD

**Issue by roomwillow** [grokability/snipe-it#8509]
#### Please confirm you have done the following before posting your bug report:

- [Y] I have enabled debug mode 
- [Y] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
Fresh deployment, accessing the webpage re-directs with a "/setup" automatically, page responds with "Not Found, the requested URL was not found on the server"

**To Reproduce**
go to the website URL

**Expected behavior**
website loads the initial setup screen

**Screenshots**

**Server (please complete the following information):**
 - Snipe-IT Version Latest
 - OS: [e.g. Ubuntu, CentOS] Debian
 - Web Server: [e.g. Apache, IIS] Apache
 - PHP Version Latest

**Error Messages**
Not Found
The requested URL was not found on this server

**Additional context**

the following is what has been done since the system was deployed, nothing more:

created new system user "snipeit" and added to sudo group, all following done from snipeit user:

installed git, ran git clone to /etc/snipe-it/

installed php-common, libapache2-mod-php php-cli php-xml php-bcmath php-zip php-ldap php-mysql php-curl php-tokenizer php-mbstring php-pdo php-openssl php-gd

installed mariadb-server

installed apache2

a2enmod rewrite (set to enabled)

copied /etc/snipe-it/.example.env to /etc/snipe-it/.env

ran mysql secure installation
sql: created user
sql: created database
sql: granted user all privs on database
sql: flushed privs

added snipeit user to www-data group
chown -R snipeit /etc/snipe-it
chmod -R 775 /etc/snipe-it/storage
chmod -R 775 /etc/snipe-it/public

curl -sS https://getcomposer.org/installer | php
php composer.phar install --no-dev --prefer-source

modified .env file with correct URL and database

php artisan key:generate
- verified the key entered properly into the env file

nano /etc/apache2/sites-available/000-default.conf
 - modified document root to /etc/snipe-it/public

nano /etc/apache2/apache2.conf
 - modified <Directory /var/www/>" to "/etc/snipe-it/public"

chgrp -R www-data /etc/snipe-it/public
chgrp -R www-data /etc/snipe-it/storage

I suspect its a permissions related issue, but ive verified the public and storage directories are recursively owned by www-data, chmod has recursively given level 7 to the group on both directories, I'm at a loss of what else I could have possibly messed up here.

**Comment by roomwillow** [comment]
closed: virtualhost was missing "AllowOverride All"

**Comment by Alex1397Pearce** [comment]
Hi, i have a problem with installation Snipe-it. Page my_domain/setup is not Found. I use snipe 6.0.9, Ubuntu 20.04, nginx. Snipe was installed with install.sh. Settings and rules are correct what i checked. Log nginx say what fno such file or directory /var/www/html/snipeit/public/setup.

**Comment by Jhonatanufcat** [comment]
I have the same problem, can anyone help me?

**Comment by sean7084** [comment]
I got the same error setting up my server. Later I found out the error was due to a Directory value (4th line) that is different from the DocumentRoot value (3rd line). Check out if you made the same mistake.


Following is the correct code:

<VirtualHost *:81>
ServerName ams2.example.com
DocumentRoot /var/www/snipe-it2/public
<Directory /var/www/snipe-it2/public>
Options Indexes FollowSymLinks MultiViews
AllowOverride All
Order allow,deny
allow from all
</Directory>
</VirtualHost>
